deploy/handler/operator.yaml

```diff
@@ -1,37 +1,35 @@
 {{define "handlerPrefix"}}{{with $prefix := .HandlerPrefix}}{{$prefix | printf "%s-"}}{{end -}}{{end}}
 ---
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
 metadata:
-  name: {{template "handlerPrefix" .}}nmstate-handler
```

Comment:
We will need to add removal code for this to CNAO. Like the one we have for multus.

Reply:
noted